product_lot_sequence/views/res_config_settings_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="stock.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='production_lot_info']" position="inside">
+            <xpath expr="//block[@id='production_lot_info']" position="inside">
                 <div class="col-12 col-lg-6 o_setting_box">
                     <div class="o_setting_right_pane">
                         <label for="lot_sequence_padding" />
```

Comment:
This needs more changes, check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0